PublicMapping/districtbuilder
# Style "Copy map" modals

## Overview
Style pass of several modal dialogs related to the new "Copy map" feature.

### Checklist


### Demo
#### When logged in - the "Copy Map" modal:
<img width="1700" alt="Screen Shot 2021-01-11 at 5 55 28 PM" src="https://user-images.githubusercontent.com/5672295/104190937-40610780-5436-11eb-8986-34ffa253c73c.png">

#### When not logged in
<img width="1695" alt="Screen Shot 2021-01-11 at 5 52 11 PM" src="https://user-images.githubusercontent.com/5672295/104190976-51117d80-5436-11eb-868a-57d11d9d2ae4.png">
<img width="1792" alt="Screen Shot 2021-01-11 at 5 45 17 PM" src="https://user-images.githubusercontent.com/5672295/104190993-5a024f00-5436-11eb-9b6a-24a849bb0ed2.png">
<img width="1792" alt="Screen Shot 2021-01-11 at 5 45 20 PM" src="https://user-images.githubusercontent.com/5672295/104191008-5ec70300-5436-11eb-8e35-752f99bf4d1c.png">


### Notes
- I was getting errors for the `AriaModal` component related to the `initialFocus` property - which I fixed by setting the same id for every primary button for each content type that populates in the `CopyMapModal` component (`#primary-button`). I don't think it fires any time other than the first time, so the "Login" and "Create an account" primary buttons are not in-focus initially... which seemed fine at the moment. Noting in case I missed anything - it's probably worthwhile to test with the console open.
- I updated the README instructions for using a proxy to suggest (what I think is) the correct URL.

## Testing Instructions
- `git pull`
- Enter any map
- If you don't see the "Copy map modal" button, go to `ProjectHeader.tsx` and copy/paste `<CopyMapButton invert={true} />` after line `129`.
- View the front-end and click "Copy this map" button.
- You should see the first modal in the screenshots. The primary button should be in-focus initially.
- Copy the page's url and open up in an incognito window.
- Now click "Copy this map" button.
- Note the styled "`So-and-so` builds maps with..." modal.  "Create a free account" should be in-focus initially.
- Click "Login" 
- Note the styled "Login" modal.
- Click "Sign up for free".
- Note the styled "Create an account" modal.

Closes #527
